packages/components/src/components/wallet-card/wallet-card.scss

```
@@ -0,0 +1,156 @@
+.wallet-card {
+    position: relative;
+
+    &__container {
+        border-radius: inherit;
+        width: 100%;
+        height: 100%;
+
+        &--active {
+            border: 2px solid var(--text-red);
+        }
+
+        &--small {
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        }
+
+        &-fade {
+            position: absolute;
+            width: 100%;
+            height: 100%;
+            top: 0;
+            left: 0;
+            opacity: 0;
+            background: var(--wallets-card-active-gradient-background);
+            border-radius: inherit;
+
+            &--active {
+                opacity: 1;
+            }
+        }
+
+        &:hover & {
+            &-fade {
+                opacity: 1;
+                &--disabled,
+                &--faded,
+                &--added {
+                    opacity: 0;
+                }
+            }
+        }
+    }
+
+    &__shine {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        clip-path: polygon(40% 10%, 104% -6.94%, 92.5% 100%, 28% 100%);
+        mix-blend-mode: overlay;
+        opacity: 0.16;
+        border-top-right-radius: $BORDER_RADIUS_2;
+        background-color: $color-white;
+    }
+
+    &__content {
+        position: relative;
+        height: 100%;
+        width: 100%;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        align-items: center;
+        padding: 1.6rem;
+
+        &--medium {
+            @include mobile {
+                padding: 0.8rem;
+            }
+        }
+    }
+
+    &--small {
+        width: 6.4rem;
+        height: 4rem;
+        border-radius: $BORDER_RADIUS;
+    }
+
+    &--medium {
+        width: 20rem;
+        height: 12rem;
+        border-radius: $BORDER_RADIUS_2;
+
+        @include mobile {
+            width: 16rem;
+            height: 9.6rem;
+        }
+    }
+
+    &--large {
+        width: 24rem;
+        height: 14.4rem;
+        border-radius: $BORDER_RADIUS_2;
+
+        @include mobile {
+            width: 21.6rem;
+            height: 12.8rem;
+        }
+    }
+
+    &__active-icon {
+        position: absolute;
+        width: 100%;
+        height: 100%;
+        top: 0;
+        left: 0;
```

Correct me if I'm wrong here but:

```suggestion
        inset: 0;
```